### PR TITLE
Gateway dev 5 2

### PR DESCRIPTION
**Pull request by jburel**
Testing the changes.
Travis builds are green.

To test:
 * checkout code
 * import maven project
 * set the credentials in the main method.
 * Run the example

possible action:
 * parse args to set credentials

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -17,19 +17,34 @@
...
         <dependency>
           <groupId>omero</groupId>
           <artifactId>blitz</artifactId>
+          <version>5.2.0-ice35-b12</version>
         </dependency>
         <dependency>
           <groupId>ome</groupId>
           <artifactId>bioformats_package</artifactId>
         </dependency>
+        <dependency>
```
Comment (joshmoore) [inline pom.xml]: Shouldn't these be pulled in by some "gateway" dependency if not "blitz" itself?

Comment (jburel) [inline pom.xml]: @joshmoore: could be, I did not modify that part. I will look into it.

File: pom.xml [hunk 2]
```diff
@@ -17,19 +17,30 @@
     <version>5.2.0</version>
```
Comment (sbesson) [inline pom.xml]: Following the same strategy a `pom-omero-client` thoughts about bumping this to 5.2.1-SNAPSHOT?

Comment (jburel) [inline pom.xml]: I think we should have one consuming the release versions
then a follow-up 5.2.1-SNAPSHOT consuming the snapshots

**Comment by jburel** [comment]
Discussed with @sbesson this am.
the plan was to already have new methods in gateway to connect via args (See https://trello.com/c/0hvmrXSL/10-gateway-improvements)
We will introduce that in this PR so version will become snapshot

**Comment by jburel** [comment]
Last commit is there to try a new workflow and will be removed when https://github.com/openmicroscopy/openmicroscopy/pull/4327 is merged

**Comment by jburel** [comment]
Remove usage of unstable snapshot now that https://github.com/openmicroscopy/openmicroscopy/pull/4327 has been merged.

**Comment by sbesson** [comment]
Works for me as https://ci.openmicroscopy.org/job/OMERO-DEV-merge-maven-client/28/ is green. Once this is in, we can come back to including https://github.com/ome/pom-omero-client/pull/7 and fixing the `pom.xml` in this repository to inherit the parent POM versions. Thoughts @joshmoore ?

**Comment by dominikl** [comment]
Works for me, too. Looks good. I actually should to the same for the pom in my R repository, point to 5.2.1 snapshot, or better also inherit from this one (still have to figure that out, how that works).

**Pull request merged**